### DISCUSSION TITLE
Beatmap forks

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#20322] [Ideas]
Example situation: 
- I created a beatmap (is not my one) by using Edit + Create new difficulty option.
- Now I can't split it, thus I can't submit it as well. (Or I can't find this "secret" option that very unlikely)
or
- I want to finish abandoned map (not mine). 

Why not to use reference to already uploaded mp3 file on server which can significantly reduce disk space occupied on server if it will be implemented?
So, beatmap forks would be a nice possibility to simplify creation of beatmaps and optimize disk space consumption.

**Comment by peppy** [comment]
The game already does this at a file system level, for what that's worth.
